### ISSUE TITLE
Machine Learning Scientist (#1108988)

### ISSUE THREAD

**Issue by PaigeMcAlister19** [tech256/jobs#421]
PeopleTec is currently seeking a Machine Learning Scientist to support our Huntsville, AL location.

The candidate will support our customers with cutting edge technologies and products for analyzing cybersecurity-related network data, along with opportunities in other event-driven problem sets involving imagery, audio and text.

The successful candidate will: mine and analyze data from network logs to drive improvement of operating methods, anomaly and event detection, and network defense strategies; evaluate the effectiveness of adversarial models used to attack existing machine learning algorithms; assess the effectiveness and accuracy of new data sources and industry-leading learning techniques; develop custom data models and algorithms to apply to data sets; use predictive modeling to increase and optimize operational outcomes; develop what-if models with A/B testing framework and test model quality; coordinate with different functional teams and stakeholders to implement models and monitor outcomes; develop processes and tools to monitor and analyze model performance and data accuracy.

 

Required Skills/Experience:

Strong problem-solving skills with an emphasis on applied machine learning
Adept at using large data sets to find opportunities for machine learning and using models to test the effectiveness of different courses of action
Coding knowledge and experience with several languages: Python, C++, Java, JavaScript, etc. to include querying databases
Passion for discovering solutions hidden in large data sets and working with stakeholders to improve decision making
Strong experience using a variety of algorithmic methods, using a variety of data tools, building and implementing models and creating/running training data
Experience using web services: Redshift, S3, Spark, etc.
Experience using log file analyzers and visualization tools such as Splunk, Elasticsearch, Logstash, and Kibana (ELK), both on premises and in the cloud Experience using statistical computer languages (R, Python, SLQ, etc.) to manipulate data and draw insights from large data sets
Knowledge and experience in advanced statistical and data mining techniques (GLM/Regression, Random Forest, Boosting, Trees, text mining, social network analysis, etc.)
Experience with applications that serve inferences from trained models
Experience creating and using advanced machine learning algorithms and techniques (simulation, scenario analysis, clustering, decision tree learning, artificial neural networks, etc.) and their real-world advantages/drawbacks
Experience working with and creating data architectures, including open source tools such as Tensorflow, Torch/PyTorch, Keras, and MxNet
Experience analyzing data from 3rd party providers: Microsoft Azure, Google APIs Experience with distributed data/computing tools: Map/Reduce, Hadoop, Hive, Spark, SQL Server, etc.
Excellent written and verbal communication skills for coordinating across teams
Drive to learn and master new technologies and techniques
Proven ability to drive decisions with their modeling insights
Travel: Up to 5% 
Must be a U.S. Citizen
An active DoD Secret clearance is required to perform this work. Candidates are required to have an active Secret clearance upon hire, and the ability to maintain this level of clearance during their employment.
 

Education Requirements:

A Bachelor's Degree, Masters, or Ph.D. in Statistics, Mathematics, Computer Science or another quantitative field with 5+ years of experience manipulating data sets and building statistical models with the skills specified above is required. 

 

Desired Skills:

 Advanced technical degree 
 

 

EOE/Minorities/Females/Veterans/Disabled

PeopleTec, Inc. is an Equal Employment Opportunity employer and provides reasonable accommodation for qualified individuals with disabilities and disabled veterans in its job application procedures. If you have any difficulty using our online system and you need an accommodation due to a disability, you may use the following email address, HR@peopletec.com and/or phone number (256.319.3800) to contact us about your interest in employment with PeopleTec, Inc.

All qualified applicants will receive consideration for employment without regard to race, color, religion, sex, sexual orientation, gender identity, national origin, age, genetic information, citizenship, ancestry, marital status, protected veteran status, disability status or any other status protected by federal, state, or local law. PeopleTec, Inc. participates in E-Verify.

For more information, or to apply now, you must go to the website below. Please DO NOT email your resume to us as we only accept applications through our website.

https://www.applicantpro.com/j/1108988-305157

**Comment by PallasStreams** [comment]
Thanks for contributing to this issue. As it has been over 90 days since the last activity, we will be closing the issue. This is often because the posting wasn't updated or it's no longer applicable. If that's not the case, please reach out to @GeneralPallas.
